model/vcs/registry.yaml

```
@@ -12,11 +12,32 @@ groups:
           The [URL](https://wikipedia.org/wiki/URL) of the repository
           providing the complete address in order to locate and identify the
           repository.
+        note: |
+          The URL SHOULD only be an HTTPS absolute URL and SHOULD NOT include
+          `.git` appended on the end nor be an SSH URL.
```

Should we add an exception for ssh if HTTPS access is not available?
Are there repo names ending with `.git`? Should we specify that the `.git` refers only to the optional suffix?
```suggestion
        note: |
          The URL SHOULD only be an HTTPS absolute URL and SHOULD NOT include
          any optional `.git` appended on the end nor be an SSH URL (unless ssh is the only protocol to access the repository).
```
Maybe it's better to split this into two separate sentences for simplicity.

---

Remark from Josh:
we should specify that this note refers to git repositories.
Other VCS types might use other protocols.

---

Yes, this URL is only for accessing the repository through a browser. This isn't the URL to clone the repository over any type of CLI utility. Regardless of the vendor, or technology (git / svn / mercurial) AFAIK, all systems have URLs of which can be hit through the browser that take you to the repository.